test/language/statements/async-generator/yield-star-promise-not-unwrapped.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2022 André Bargull. All rights reserved.
```

Comment:
Is this test intentionally attributed to André?

Reply:
Whoops, no, I just forgot to update it when I copied an existing test.